### ISSUE TITLE
Python 3.9: TypeError: Invalid first argument to `register()`. ForwardRef('CachedResponse') is not a class.

### ISSUE THREAD

**Issue by a0x8o** [mgedmin/cloneall#20]
On Python 3.9 I observe the following error (macOS):
```
File "/usr/local/Cellar/python@3.9/3.9.15/Frameworks/Python.framework/Versions/3.9/lib/python3.9/functools.py", line 855, in register
    raise TypeError(
TypeError: Invalid first argument to `register()`. ForwardRef('CachedResponse') is not a class.
```

**Comment by mgedmin** [comment]
Can you show me the full traceback please?

**Comment by mgedmin** [comment]
Google tells me this might be a bug in one of ghcloneall's dependencies, requests-cache: https://github.com/requests-cache/requests-cache/issues/501

It was reported as fixed in requests-cache 0.9.1.  What version of requests-cache do you have installed?

**Comment by mgedmin** [comment]
As a workaround perhaps you can use `ghcloneall --no-http-cache`?  The HTTP cache is primarily useful for development, when you run ghcloneall multiple times in succession, to avoid hitting GitHub API rate limits for unauthenticated requests.

Since I don't have the full traceback I can't tell if this happens at import time, or during `install_cache()`.  If it happens during import time, `--no-http-cache` will not help.
